DOCS-10828 teams (June 12) (#136)
mulesoft/docs-exchange: modules/ROOT/pages/to-configure-api-settings.adoc

endif::[]
:imagesdir: ../assets/images/

> Use Anypoint Exchange to configure whether API instances are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.

Reviewer: ```suggestion
Use Anypoint Exchange to configure whether API instances are public or private in a private Exchange server or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.
```
GLOBALLY avoid using trademarks as nouns preceded by articles (a, the); provide a common noun object after them, making them adjectives, per style ref. and trademarking rules.

Author: Good point! Fixed.

After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API instances list in the navigation bar.

> Exchange also creates API instances for APIs managed by another API management program outside of Anypoint Platform.

Reviewer: ```suggestion
Exchange also creates API instances for APIs managed by another API management program besides Anypoint Platform.
```

Author: Fixed.

Visibility for each instance is either private or public. Users can see a private instance if the asset is shared with them or if they have an Anypoint Platform permission that grants access. All users accessing an Exchange public portal can see all public API instances. All instances in an Exchange public portal are read only.

> In a private Exchange, asset administrators have access to change the visibility of API instances, and to add, edit, and delete instances.

Reviewer: ```suggestion
In a private Exchange server, asset administrators have access to change the visibility of API instances, and to add, edit, and delete instances.
```
portal? GLOBALLY provide a common noun following Exchange.

Author: Fixed.

endif::[]
:imagesdir: ../assets/images/

Use Anypoint Exchange to configure whether REST, SOAP, and HTTP API services are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.

After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API instances list in the navigation bar.

Exchange also creates API instances for APIs managed by another API management program outside of Anypoint Platform.

Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs. All instances in an Exchange portal are read only.

Visibility for each instance is either private or public. All users in a business group's Exchange portal can see a public instance. Users can see a private instance if the asset is shared with them or if they have an Anypoint Platform permission that grants access.

> In a private Exchange, asset administrators have access to change the visibility of API instances and add new instances. Make an instance public with the visibility indicator. Edit the instance with the pencil icon. Delete the instance with the X icon.

Reviewer 2: Bulleted list would work well here.

Author: That is better! Updated.

endif::[]
:imagesdir: ../assets/images/

Use Anypoint Exchange to configure whether REST, SOAP, and HTTP API services are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.

After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API instances list in the navigation bar.

Exchange also creates API instances for APIs managed by another API management program outside of Anypoint Platform.

> Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs. All instances in an Exchange portal are read only.

Reviewer 3: > Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs

I'm not sure when did we add this, but it's not accurate. The API instances list appear for HTTP, REST and SOAP APIs in both private Exchange and Exchange Public Portals (also I assume that "Exchange portal" refers to "Exchange Public Portal" in this context, let's make it more explicit by using the latter term). I don't think there is a need to call this out.

>All instances in an Exchange portal are read only.

This is accurate 👍  (assuming we replace "Exchange portal" with "Exchange public portal"). This phrase could be moved after the following paragraph that discusses API Instance visibility.

Author: Thank you! Fixed.

> Visibility for each instance is either private or public. All users in a business group's Exchange portal can see a public instance. Users can see a private instance if the asset is shared with them or if they have an Anypoint Platform permission that grants access.

Reviewer 3: >All users in a business group's Exchange portal can see a public instance

The wording here seems off since Exchange Public Portals are not scoped to Bussiness Groups. We could change this to
"All users accessing an Exchange Public Portal can see a public API instance."

Author: Thanks! Fixed.
